app/src/app.js

```diff
@@ -33,6 +33,13 @@ if (document.querySelector('.index')) {
     return update('index');
   });
 
+  socket.on('START_GAME', messageData => {
+    console.log(data[messageData.gameID].playing)
+    data[messageData.gameID].playing === true;
+    console.log(data[messageData.gameID].playing)
+    return update('index', data);
```

Comment:
Can you take a look at what I tried to do with that, first? I wanted to give the rooms that are currently playing a class, so new players can see which game to spectate in the lobby.

The `data[messageData.gameID.playing === true` doesn't do anything, since both consoles output false. But since I started writing this comment I noticed I should've used `=`, instead of `===` 💯 

I left those logs in there on purpose, but should've commented it out, sorry.